portal/templates/admin/user-edit.njk

```diff
@@ -92,6 +92,13 @@
           checked: displayedUser.roles.includes("checker"),
           attributes: {"data-cy": "role-checker"}
         },
+        {
+          value: "payment-officer",
```

Comment:
it would be nicer if we passed these values in from the constants in the NJK files but given read-only work didn't do this I'm guessing it was decided against so let's leave it as this, I just like leaving random thoughts on PRs apparently